src/common/Defines.cpp

```diff
@@ -143,6 +143,15 @@ DEFINE_double(
     100,
     "percentage of training set (by number of utts) to use for evaluation");
 
+// Mixed Precision OPTIONS
+DEFINE_bool(mixedprecision, false, "Use mixed precision for training");
+DEFINE_int32(scalefactor, 4096, "Loss scale factor");
+DEFINE_int32(
+    scalefactorupdateinterval,
```

Comment:
See my comment above. If FP16 can work well-enough without this scaling, I'd prefer if we merge the needed changes as they are, then focus on changes needed to make those models converge better.